ospo-mindmap/Content/ospomindmap.md

```diff
@@ -88,6 +88,7 @@ development team of the Linux Project
 - Marketing
 - Communications
 - Growing and retaining open source talent
+- Maintain relationships with communities of strategic importance to your organization
```

Comment:
If we start elaborating the execution part of the strategy then "Maintain relationships..." should be a sub bullet under Open Source community, contributions and leadership